tests/common.js

```diff
@@ -46,6 +54,9 @@ function loadQml(src, opts) {
                   ' is known to be failing. Skipping...');
       return;
     }
+    var div = document.createElement('div');
+    this.div = div;
     itOrig.apply(this, arguments);
+    div.remove();
```

Comment:
Aliasing is useful where the variable is used multiple times, but here it's just created and removed.